### ISSUE TITLE
Fiddling with hyperparameters

### ISSUE THREAD

**Issue by dugongdingo** [dugongdingo/WEEL#6]
Here is a non-inclusive list of parameters for which different values should be tested:

- learning rate
- dropout
- number of epochs
- encoder and decoder optimizers

Both a grid search and a random search are possible, as the model is (for now) simple enough for the parameters to be exhaustively tested.

**Comment by dugongdingo** [comment]
in 59bb78d5d7f13070a0c2b9c1df1c5da0b2b86c8b : testing parameters learning rate & dropout using exponential steps. epochs are fixed to 5
